definitions/declarations/dfe-analytics-dataform/declare_npq_dfe_analytics_dataform_tables.js

```diff
@@ -15,7 +15,9 @@ const tableNames = [
     "statement_items_latest",
     "statements_latest",
     "users_latest",
-    "users_version"
+    "users_version",
+    "delivery_partners_latest",
+    "delivery_partnerships_latest"

```

Comment:
@t-phillips-dfe you don't seem to be using delivery_partnerships_latest anywhere. Suggest removing the declaration as it'll only clog up the DAG.